flake.nix

```diff
@@ -3,14 +3,68 @@
 
   inputs = {
     flake-utils.url = "github:numtide/flake-utils";
-    nixpkgs.url = "nixpkgs/21.11";
+    nixpkgs.url = "nixpkgs/release-22.11";
+    poetry2nix = {
+      url = "github:nix-community/poetry2nix";
+      inputs = {
+        nixpkgs.follows = "nixpkgs";
+        flake-utils.follows = "flake-utils";
```

Comment:
Here I made `poetry2nix` use the same version of `flake-utils` as for our flake. Now we have:

```sh-session
$ nix flake metadata
Resolved URL:  git+file:///home/steshaw/Code/tweag/chainsail
Locked URL:    git+file:///home/steshaw/Code/tweag/chainsail?ref=refs%2fheads%2fupgrade-poetry&rev=65919a7d89c4aa4b2b606494224eb7b6a01a64c9
Description:   Chainsail
Path:          /nix/store/jcsbg1rj57mspknnc67w8vr0gmz9dkhz-source
Revision:      65919a7d89c4aa4b2b606494224eb7b6a01a64c9
Revisions:     418
Last modified: 2023-03-08 10:35:41
Inputs:
├───flake-utils: github:numtide/flake-utils/74f7e4319258e287b0f9cb95426c9853b282730b
├───nixpkgs: github:NixOS/nixpkgs/96e18717904dfedcd884541e5a92bf9ff632cf39
└───poetry2nix: github:nix-community/poetry2nix/45babaf3f04aa1d53fa90e7ab9404360f7621919
    ├───flake-utils follows input 'flake-utils'
    └───nixpkgs follows input 'nixpkgs'
```

Reply:
Perfect, thanks!